akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/TimeoutDirectivesExamplesSpec.scala

```diff
@@ -25,9 +25,7 @@ private[this] object TimeoutDirectivesTestConfig {
     akka.stdout-loglevel = ERROR
     windows-connection-abort-workaround-enabled = auto
     akka.log-dead-letters = OFF
-    akka.http.server.request-timeout = 1000s""")
-  // large timeout - 1000s (please note - setting to infinite will disable Timeout-Access header
-  // and withRequestTimeout will not work)
+    akka.http.server.request-timeout = infinite""")
```

Comment:
Before this PR, setting the `request-timeout` to `infinite` would result in some of the tests in `TimeoutDirectivesExamplesSpec` failing. In order to pass the timeout would have to be set to any finite value. Now the tests pass, independent of whether the `request-timeout` is finite (e.g. `1000s`) or `infinite`.